include/oneapi/dnnl/dnnl_types.h

```diff
@@ -2002,7 +2002,12 @@ typedef enum {
 /// A `size_t` counterpart of the DNNL_RUNTIME_DIM_VAL.
 /// For instance, this value is returned by dnnl_memory_desc_get_size() if
 /// either of the dimensions or strides equal to #DNNL_RUNTIME_DIM_VAL.
+
+#ifdef DNNL_X64
 #define DNNL_RUNTIME_SIZE_VAL ((size_t)DNNL_RUNTIME_DIM_VAL)
+#else
+#define DNNL_RUNTIME_SIZE_VAL ((size_t)INT32_MIN)
```

Comment:
I believe we should keep the value unchanged for AARCH64 as well

Reply:
fixed